src/components/Screen.tsx

```diff
@@ -79,6 +79,7 @@ export class InnerScreen extends React.Component<ScreenProps> {
       // Due to how Yoga resolves layout, we need to have different components for modal nad non-modal screens
       const AnimatedScreen =
         Platform.OS === 'android' ||
+        stackPresentation === undefined ||
```

Comment:
Wouldn't it be better to do something like `stackPresentation === 'modal' ? AnimatedNativeModalScreen : AnimatedNativeScreen` - use modal when necessary and regular screen for rest, instead of specifying all the cases where it won't be a modal?

Reply:
This check will probably be updated when modals on `Android` and other types of modals on `iOS` by @kkafar will be merged. I'd leave it like that for now and come back to it when it's done but if you feel like it should be done here, then we can do it 🤷‍♂️